embedded-nal-async/Cargo.toml

```diff
@@ -11,6 +11,9 @@ readme = "README.md"
 keywords = ["network"]
 categories = ["embedded", "hardware-support", "no-std", "network-programming", "asynchronous"]
 
+[features]
+ip_in_core = []
```

Comment:
Can't we get rid of no_std_net in this case?

Reply:
> Can't we get rid of no_std_net in this case?

Good point! I made it an optional feature.